src/dsp/ethereum/aux.ts

```diff
@@ -21,6 +22,17 @@ export function genAuxParams(
 }
 
 function fillMPTInput(input: any, _cleYaml: CLEYaml, _dataPrep: EthereumDataPrep) {
+  let mptIpt = new MptInputPrep(_dataPrep.blocknumberOrder.length)
```

Comment:
fyi: all data prepare process should stay in "prepare()" stage, "fill" should keep as a "pure/view" function. Although, this MptInputPrep is not a result of data prepare process, so it's not here but the class name needs to be changed.